src/rsa/bigint.rs

```diff
@@ -67,7 +67,7 @@ struct Width<M> {
 }
 
 /// All `BoxedLimbs<M>` are stored in the same number of limbs.
-struct BoxedLimbs<M> {
+pub struct BoxedLimbs<M> {
```

Comment:
Let's make the primality test code a submodule of `bigint` so that we don't need to increase the visibility of things in `bigint`.